README.md

```
@@ -1,3 +1,53 @@
+  - [ ] 장군 여부에 따른 이동 가능 여부 검증
+- [3] 장기 말 경로 내비게이터
+  - [x] 시작 좌표와 도착 좌표 사이의 경로 좌표 반환
+- [3] 말
+  - [x] 위치 이동
+  - [x] 차지 여부 확인
+  - [ ] 생사 여부 확인
+  - [ ] 이동 규칙 저장
+- [4] 위치
+  - 좌표 값(x, y) 저장
+
+- 테스트는 향후 추가할 예정입니다.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)